src/game.cpp

```diff
@@ -12629,7 +12639,17 @@ void set_scenario( const scenario *new_scenario )
     g->scen = new_scenario;
 }
 
+scent_map &get_scent()
+{
+    return g->scent;
+}
+
 stats_tracker &get_stats()
 {
     return g->stats();
 }
+
+timed_event_manager &get_timed_events()
```

Comment:
what think you about putting these in a namespace, like `cata::` or somesuch?

Reply:
Yea as the number of them has started piling up I've been pondering that.